### PR TITLE
Add German Quotes

### PR DESCRIPTION
**Pull request by meyerhenning**
Adding german quotes 563 - 575

### CODE REVIEW

**Comment by Miodec** [comment]
Could you translate them so I can verify content?

**Comment by Miodec** [comment]
Also please resolve merge conflicts

**Comment by meyerhenning** [comment]
Weird conflicts but yeah, resolved.

Well, I could translate the quotes (more or less), but this would double the effort for a simple change.
I mean, I understand your concerns, but translating all the quotes would be quite a hassle.
Especially when I think about additional PRs in the future.

**Comment by Miodec** [comment]
Its the only way to verify you're not adding anything extremely offensive.

File: frontend/static/quotes/german.json [hunk 1]
```diff
@@ -3326,10 +3326,88 @@
       "id": 561
     },
     {
-      "text": "Es ist besser, ein einziges kleines Licht anzuzünden, als die Dunkelheit zu verfluchen.",
-      "source": "Konfuzius",
-      "length": 87,
-      "id": 562
+	    "text": "Es ist besser, ein einziges kleines Licht anzuzünden, als die Dunkelheit zu verfluchen.",
```
Comment (meyerhenning) [inline frontend/static/quotes/german.json:3329]: "It's better to put on a light than get annoyed by the darkness."

**Comment by meyerhenning** [comment]
Added translations

**Comment by Miodec** [comment]
> Added translations

I can only see one of them. How did you add them?

**Comment by meyerhenning** [comment]
I added a comment at each quote.
Seems like they are part of a review and are "Pending".
Is there a way to to publish them? Do I have to submit my review or something?

**Comment by Miodec** [comment]
Alright, thanks

**Pull request merged**